### PR TITLE
Development

### PR DESCRIPTION
**Pull request by brunovasquez**
WildCards

### CODE REVIEW

**Comment by brunovasquez** [comment]
@carledriss ready for code review

File: src/test/java/org/fundaciojala/org/WildsTest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package org.fundaciojala.org;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+/**
+ * Test do {@link SearchEngine}
+ */
+public class WildsTest {
+    private String haystack = "Once upon a midnight dreary, while I pondered, weak and weary";
```
Comment (carledriss) [inline src/test/java/org/fundaciojala/org/WildsTest.java:11]: this should be a constant
private static final

File: src/main/java/org/fundaciojala/org/SearchEngine.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    /**
+     * Finds and fill a word in a sentence
+     *
+     * @param word string to search.
+     * @param haystack string, the sentence to find the word
+     * @return return true or false
+     */
+    static int find(String word, String haystack) {
+        String regWord = "\\Q" + word.replaceAll("_", "\\\\E.\\\\Q") + "\\E";
```
Comment (carledriss) [inline src/main/java/org/fundaciojala/org/SearchEngine.java]: This should be final
Please use concat method instead of +

**Comment by carledriss** [comment]
@brunovasquez 
Let me know when you have fixed the observations

**Comment by brunovasquez** [comment]
@carledriss 
Ready to Code Review.

**Comment by carledriss** [comment]
Looks good to me!

**Pull request merged**